dev/export-vault.ctmpl

```diff
@@ -1,23 +1,48 @@
 #!/usr/bin/env bash
 set -e
 
+function log_deprecated(){
+    path="${1}"
+    env="${SERVICE_ENV}"
+    product="${SERVICE_PRODUCT}"
+    service="${SERVICE_NAME}"
+    (>&2 echo "{\"level\":\"warn\",\"path\":\"${path}\",\"env\":\"${env}\",\"product\":\"${product}\",\"msg\":\"loaded deprecated value\"}")
+}
+
 # DEPRECATED: global secrets
 {{range secrets (printf "secret/global/%s/env_vars" (env "SERVICE_ENV")) -}}
 {{if not (env (. | toUpper)) -}}
 export {{ . | toUpper}}{{with secret (printf "secret/global/%s/env_vars/%s" (env "SERVICE_ENV") .) }}="{{.Data.value}}"{{end}}
+log_deprecated "secret/global/${SERVICE_ENV}/env_vars/{{.}}"
+{{end -}}
+{{end -}}
+
+# global secrets
+{{range secrets "secret/global/env_vars" -}}
+{{if not (env (. | toUpper)) -}}
+export {{ . | toUpper}}{{with secret (printf "secret/global/env_vars/%s" .) }}="{{.Data.value}}"{{end}}
 {{end -}}
 {{end -}}
 
 # DEPRECATED: products secrets
 {{if (env "SERVICE_PRODUCT") -}}
 {{range secrets (printf "secret/products/%s/dev/env_vars" (env "SERVICE_PRODUCT")) -}}
```

Comment:
Noticed this one had a "dev" hardcoded for env. Not exactly sure why. (peer->stage makes sense, but wasn't sure about dev)

Reply:
Not sure why either. dev is always dev, so this is fine to leave as-is since it is deprecated